Author: Saleem Abdulrasool <compnerd@compnerd.org>
Upstream: No - exherbo specific
Reason: 32-bit libraries (including the dynamic linker) reside in /lib32
--- 
--- gcc-4.4.1/gcc/config/i386/linux.h	2009-09-12 17:54:22.800792942 +0000
+++ gcc-4.4.1/gcc/config/i386/linux.h	2009-09-12 17:54:35.095793260 +0000
@@ -100,7 +100,7 @@
 
 /* These macros may be overridden in k*bsd-gnu.h and i386/k*bsd-gnu.h. */
 #define LINK_EMULATION "elf_i386"
-#define GLIBC_DYNAMIC_LINKER "/lib/ld-linux.so.2"
+#define GLIBC_DYNAMIC_LINKER "/lib32/ld-linux.so.2"
 
 #undef  ASM_SPEC
 #define ASM_SPEC \
--- gcc-4.4.1/gcc/config/i386/linux64.h	2009-09-12 17:54:25.820793057 +0000
+++ gcc-4.4.1/gcc/config/i386/linux64.h	2009-09-12 17:55:03.739793298 +0000
@@ -58,7 +58,7 @@
    When the -shared link option is used a final link is not being
    done.  */
 
-#define GLIBC_DYNAMIC_LINKER32 "/lib/ld-linux.so.2"
+#define GLIBC_DYNAMIC_LINKER32 "/lib32/ld-linux.so.2"
 #define GLIBC_DYNAMIC_LINKER64 "/lib64/ld-linux-x86-64.so.2"
 
 #undef ASM_SPEC
